### ISSUE TITLE
Scrobble a song more than once

### ISSUE THREAD

**Issue by elamperti** [elamperti/OpenWebScrobbler#128]
**Is your feature request related to a problem? Please describe**
Sometimes users want to scrobble a song more than once, either because they've listened to it several times or just because they are fixing their libraries.

**Describe the solution you'd like**
There should be an option to set how many times a track must be scrobbled.

**Additional context**
Suggested by _Butter Nuffer_ on Discord ([permalink](https://discordapp.com/channels/474201109980905483/474203420358737922/750204429562675261)).

**Comment by HighnessAtharva** [comment]
Yes, I would very much like this feature. A counter can be added next to the album/single name to which user can input a numeric value say x10. This album/single should be scrobbled 10 times with a pause of 5 seconds between each scrobbling so as to be lenient with the API calls. Would definitely be lovely to see this implemented.
